test/sanity/validate-modules/README.rst

```diff
@@ -19,53 +19,137 @@ Help
+| 403     | Type comparison using ``type()`` found. Use ``isinstance()`` instead                                                                       |
++---------+--------------------------------------------------------------------------------------------------------------------------------------------+
++---------+--------------------------------------------------------------------------------------------------------------------------------------------+
+| **5xx** | **Naming**                                                                                                                                 |
++---------+--------------------------------------------------------------------------------------------------------------------------------------------+
+| 501     | Official Ansible modules must have a ``.py`` extension for python modules or a ``.ps1`` for powershell modules                             |
++---------+--------------------------------------------------------------------------------------------------------------------------------------------+
+| 502     | Ansible module subdirectories must contain an ``__init__.py``                                                                              |
++---------+--------------------------------------------------------------------------------------------------------------------------------------------+
+| 503     | Missing python documentation file                                                                                                          |
++---------+--------------------------------------------------------------------------------------------------------------------------------------------+
 
 Warnings
 ^^^^^^^^
 
-#. Try/Except ``HAS_`` expression missing
-#. Missing ``RETURN`` for existing modules
-#. ``import json`` found
-#. Module contains duplicate globals from basic.py
-
-Module Directories (Python Packages)
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-#. Missing ``__init__.py``
++---------+--------------------------------------------------------------------------------------------------------------------------------------------+
+| code    | sample message                                                                                                                             |
++=========+============================================================================================================================================+
+| **2xx** | **Imports**                                                                                                                                |
++---------+--------------------------------------------------------------------------------------------------------------------------------------------+
+| 201     | Try/Except ``HAS_`` expression missing                                                                                                     |
```

Comment:
Yes, that is true.  I just didn't prefix everything.  The idea is that errors and warnings would have implicit E/W added.

Although something to note here, is that at least for 312 it is meant to match the error, as the condition is dependent on whether the module is new or not.